server/@types/activities.ts

```diff
@@ -72,15 +72,6 @@ export type ActivitiesFilters = {
   locationFilters: FilterItem[]
 }
 
-export type AttendanceSummaryFilters = {
-  activityDate: Date
-  searchTerm: string
-  categories: string[]
-  categoryFilters: FilterItem[]
-  activities: string[]
-  activityFilters: FilterItem[]
-}
-
 export type SubLocationCellPattern = {
```

Comment:
Have the attendance stats filters been removed? Is this by agreement with design/content/BAs?